bigquery/google/cloud/bigquery/magics.py

```diff
@@ -433,7 +455,9 @@ def _cell_magic(line, query):
 
     error = None
     try:
-        query_job = _run_query(client, query, job_config)
+        query_job = _run_query(
+            client, query, job_config=job_config, max_results=max_results
```

Comment:
`max_results` is not actually needed here, it's not until the call to `to_dataframe` that we need `max_results`.

Note: `to_dataframe` probably doesn't have a `max_results` argument, and I'm actually not certain that we'd want to add one. Instead, we can call `query_job.results` with a `max_results` argument and then call `to_dataframe` on the resulting `RowIterator`.

Reply:
I updated to call `to_dataframe` on `query_job.result`, passing `max_results` as an argument if `max_results` is present in f88ffe8